contracts/GroupHealth.sol

```diff
@@ -0,0 +1,297 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import "./common/UsingRegistryUpgradeable.sol";
+import "./common/UUPSOwnableUpgradeable.sol";
+
+import "./interfaces/IManager.sol";
+import "./interfaces/IStakedCelo.sol";
+import "./interfaces/IAccount.sol";
+import "./interfaces/ISpecificGroupStrategy.sol";
+import "./interfaces/IDefaultStrategy.sol";
+
+contract GroupHealth is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
```

Comment:
Brief NatSpec for the contract? For others too if they're missing, don't remember if I checked.